samples/templates/default-azuredeploy-docker-sql.json

```diff
@@ -415,6 +427,54 @@
                 "principalId": "[reference(concat('Microsoft.Web/sites/', variables('serviceName')), '2018-11-01', 'full').identity.principalId]",
                 "principalType": "ServicePrincipal"
             }
+        },
+        {
+            "name": "[variables('azureContainerRegistryName')]",
+            "type": "Microsoft.ContainerRegistry/registries",
+            "apiVersion": "2019-12-01-preview",
```

Comment:
>2019-12-01-preview [](start = 27, length = 18)

Just curious, is worth to use preview version here?
[https://docs.microsoft.com/en-us/azure/templates/Microsoft.ContainerRegistry/2019-05-01/registries](https://docs.microsoft.com/en-us/azure/templates/Microsoft.ContainerRegistry/2019-05-01/registries)
vs
[https://docs.microsoft.com/en-us/azure/templates/Microsoft.ContainerRegistry/2019-12-01-preview/registries](https://docs.microsoft.com/en-us/azure/templates/Microsoft.ContainerRegistry/2019-12-01-preview/registries)
looks like main difference is Identity object which is not used.

Reply:
Thanks. I have changed to the release version "2009-05-01". I just took the latest template from the documentation without noting it's preview.